xblock/core.py

```diff
@@ -174,6 +174,15 @@ def validate(self):
         """
         return Validation(self.scope_ids.usage_id)
 
+    def ugettext(self, text):
```

Comment:
@mattdrayer what you suggest if we keep the name of this function as `_` instead of `ugettext` ?